src/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.md

```diff
@@ -103,12 +103,12 @@ Rule ID | Title | Category | Enabled | Severity | CodeFix | Description |
 [CA1832](https://docs.microsoft.com/visualstudio/code-quality/ca1832) | Use AsSpan or AsMemory instead of Range-based indexers when appropriate | Performance | True | Warning | True | The Range-based indexer on array values produces a copy of requested portion of the array. This copy is usually unnecessary when it is implicitly used as a ReadOnlySpan or ReadOnlyMemory value. Use the AsSpan method to avoid the unnecessary copy. |
 [CA1833](https://docs.microsoft.com/visualstudio/code-quality/ca1833) | Use AsSpan or AsMemory instead of Range-based indexers when appropriate | Performance | True | Warning | True | The Range-based indexer on array values produces a copy of requested portion of the array. This copy is often unwanted when it is implicitly used as a Span or Memory value. Use the AsSpan method to avoid the copy. |
 [CA1834](https://docs.microsoft.com/visualstudio/code-quality/ca1834) | Consider using 'StringBuilder.Append(char)' when applicable. | Performance | True | Warning | True | 'StringBuilder.Append(char)' is more efficient than 'StringBuilder.Append(string)' when the string is a single character. When calling 'Append' with a constant, prefer using a constant char rather than a constant string containing one character. |
+[CA1835](https://docs.microsoft.com/visualstudio/code-quality/ca1835) | Prefer 'ReadOnlyMemory<Byte>' overload for 'WriteAsync'. | Performance | True | Warning | False | All 'Stream' based classes provide a 'WriteAsync' overload that takes a 'ReadOnlyMemory<Byte>' as first argument. When possible, prefer the overload that takes a 'ReadOnlyMemory<Byte>', which is more efficient. |
```

Comment:
Why is this being added in this PR?  Was it just missing erroneously?

Reply:
This was me just running `/t:pack` locally, which runs the tool to auto-update the SARIF and md files to reflect the current state. I can revert these changes and have them done in a separate cleanup PR later on.